components/sst_importer/src/metrics.rs

```diff
@@ -127,4 +127,8 @@ lazy_static! {
         &["type"],
         exponential_buckets(0.01, 4.0, 8).unwrap()
     ).unwrap();
+    pub static ref IMPORT_DENIED_REQUESTS: IntCounter = register_int_counter!(
```

Comment:
What about reuse `IMPORTER_ERROR_VEC` since deny is expected to happen very rarely.